packages/react/src/hooks/__tests__/useAuth.test.ts

```diff
@@ -54,7 +54,9 @@ describe('useAuth', () => {
 
     const { result, waitForNextUpdate } = renderHook(() => useAuth());
 
-    await waitForNextUpdate();
+    await act(async () => {
+      await waitForNextUpdate();
+    });
```

Comment:
Non-blocking but we can probably remove the `act` and `await` here if we remove the `async` declaration on line 52 (same with line 69)

Reply:
I tried that but then the result comes back undefined and the test fails